meerkat/datapanel.py

```diff
@@ -729,6 +734,85 @@ def merge(
             suffixes=suffixes,
             validate=validate,
         )
+    
+    def sort(
+        self,
+        by: Union[str, List[str]],
+        ascending: Union[bool, List[bool]] = True,
+        kind: str = "quicksort",
+    ) -> DataPanel:
+        """ 
+        Sort the DataPanel by the values in the specified columns. Similar to 
+        ``sort_values`` in pandas.
+
+        TODO(Hannah): Implement this method – should likely just wrap 
+        meerkat.ops.sort.sort. 
+
+        Args:
+            by (Union[str, List[str]]): The columns to sort by.
+            ascending (Union[bool, List[bool]]): Whether to sort in ascending or 
+                descending order. If a list, must be the same length as `by`.Defaults 
+                to True.
+            kind (str): The kind of sort to use. Defaults to 'quicksort'. Options 
+                include 'quicksort', 'mergesort', 'heapsort', 'stable'.
+
+        Return:
+            DataPanel: A sorted view of DataPanel.
+
+
+        """
+
+        panda_col_type = '<class \'meerkat.columns.pandas_column.PandasSeriesColumn\'>'
+        numpy_col_type = '<class \'meerkat.columns.numpy_column.NumpyArrayColumn\'>'
+        tensor_col_type = '<class \'meerkat.columns.tensor_column.TensorColumn\'>'
+
+        keys = []
+        
+
+        if len(by) > 1:  # Sort with multiple column
+            tensor_col_name = "none"
+            panda_col_name = "none"
+            for col in by:
+                
+                curr_col_type = str(type(self[col]))
+                print(curr_col_type)
+                # Convert all columns to numpy type
+                if curr_col_type == panda_col_type:
```

Comment:
Easiest way to check a column type is: 
```
isinstance(self[col], PandasSeriesColumn)
```
https://docs.python.org/3/library/functions.html#isinstance

This has the added advantage of also working for subclasses of PandasSeriesColumn

https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance#:~:text=answers%2C%20isinstance%20caters%20for%20inheritance,of%20subtypes%2C%20AKA%20subclasses).